File: cinema-reservations-api/Model/Movie.cs
using System.ComponentModel.DataAnnotations;

namespace cinema_reservations_api.Model {
    public class Movie {
        public int MovieId { get; set; }
        public string Title { get; set; }
        public int Duration { get; set; }

        public override string ToString() {
            return $"{MovieId} {Title} {Duration}";
        }
    }
}

File: cinema-reservations-front/lib/model/movie.dart
import 'package:json_annotation/json_annotation.dart';

part 'movie.g.dart';

@JsonSerializable()
class Movie {
  final int movieId;
  final String title;
  final int duration;

  Movie({
    required this.movieId,
    required this.title,
    required this.duration,
  });

  factory Movie.fromJson(Map<String, dynamic> json) => _$MovieFromJson(json);

  Map<String, dynamic> toJson() => _$MovieToJson(this);
}
